themes/bootstrap5/templates/RecordDriver/SolrOverdrive/previewlink.phtml

```diff
@@ -1,5 +1,5 @@
 <?php
 $previews = $this->driver->getPreviewLinks();
-if ($previews->url): ?>
+if (isset($previews->url)): ?>
```

Comment:
I think there's a bigger problem here, because `getPreviewLinks` always returns an array, so `$previews->url` is never going to be set. I wasn't sure what was intended, though, so this was my temporary fix to avoid fatal errors. I'm pretty sure more work is needed!

Reply:
I see that the function doc says it returns an array, and `links` implies plural, but it doesn't; it returns a struct with properties source, formatType and url.

![image](https://github.com/user-attachments/assets/cb1178c8-a921-49ca-8a41-d7997d936d09)

Reply:
In that case, getPreviewLinks may need to be renamed and/or have its doc comment revised. Do you have any suggestions, or should we wait and see what @bpalme says?